### DISCUSSION TITLE
Route von kurviger wird bei kleiner Änderung komplett verändert

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#239] [Q&A] [answered]
Wenn ich eine kurviger Route importieren wird sie 1:1 übernommen. Die Route wird in rosa angezeigt.
Wenn ich nur eine ganz kleine Änderung vornehme ( zusätzlicher Wp. ) wird die ganze Route komplett verändert. 
Meiner Meinung dürfte sich die Route maximal bis zum nächsten Wp./ Shapingpunkt verändern.
<img src="https://github.com/devemux86/cruiser/assets/124886589/40c70f45-a606-4eeb-ae27-1809b2095472" width="50%"/>

**Comment by 0709wiwiwi** [comment]
Planned see: https://github.com/devemux86/cruiser/discussions/142

**Comment by devemux86** [comment, marked as answer]
> Meiner Meinung dürfte sich die Route maximal bis zum nächsten Wp./ Shapingpunkt verändern.

The routes do not change in the navigation. This will also be available in the route planning:
Die Routen ändern sich in der Navigation nicht. Sie wird auch in der Routenplanung verfügbar sein:
- https://github.com/devemux86/cruiser/discussions/142

@1maxi2
Have you selected BRouter (offline)? With which options?
Haben Sie BRouter (offline) gewählt? Mit welchen Optionen?

---

You should all start using BRouter (offline). We can get similar results and everything works **offline**.
Cruiser we will soon have new BRouter profiles with new interesting options. :slightly_smiling_face: 

Sie sollten alle anfangen, BRouter (offline) zu benutzen. Wir können ähnliche Ergebnisse erzielen und alles funktioniert **offline**.
Cruiser werden wir bald neue BRouter-Profile mit neuen interessanten Optionen haben. :slightly_smiling_face: 

<img src="https://github.com/devemux86/cruiser/assets/3484020/86ebbb9d-595d-4863-a4e3-e139824b5988" width="50%"/>

**Reply by 1maxi2** [reply]
Ja brouter offline wird verwendet.
Einstellung siehe Anhang
<img src="https://github.com/devemux86/cruiser/assets/124886589/39f5e895-fa6f-47b9-a176-ef81cf5a1be6" width="50%"/>

**Reply by devemux86** [reply]
@1maxi2 

You can also try other options, like "Avoid primary roads" or "Avoid secondary roads".

Sie können auch andere Optionen ausprobieren, wie "Hauptstraßen vermeiden" oder "Sekundärstraßen vermeiden".

**Reply by 1maxi2** [reply]
Wenn ich in kurviger ein Overlay
 ( Track ) habe und ich möchte in Cruiser davon eine Route haben die genau mit dem Overlay übereinstimmt wie muss ich vorgehen ?
Ein Overlay kann ich ja nicht speichern und auch nicht teilen/ verschicken.
Der Grund ist das ich in kurviger viele schöne Overlays ( Track ) habe die ich
In Cruiser importieren möchte und dort als Route abzufahren

**Reply by devemux86** [reply]
@1maxi2 

You can import Kurviger routes in Cruiser:
- As routes for navigation
- Or overlays for route planning with BRouter (offline)

Select Route or Overlay in the import dialog. Is this what you mean?

---

Sie können Kurviger-Routen in Cruiser importieren:
- Als Routen für die Navigation
- Oder Overlays für die Routenplanung mit BRouter (offline)

Wählen Sie Route oder Overlay im Importdialog. Ist es das, was Sie meinen?

**Reply by 1maxi2** [reply]
Wenn ich aber die Tracks/ Overlays aber auf einem anderen /alten Telefon habe kann ich sie nicht verschicken 
( Keine Route vorhanden )

Aber sie sollten sich darüber nicht zu viele Gedanken machen.
Ich werde mir die Overlays am alten Telefon anschauen und in Cruiser am neuen Telefon nachzeichnen/ nachplanen
